study.md

```diff
@@ -23,4 +23,15 @@ In your own words, describe why someone would use Ember.
 
 ```md
 <!-- your answer here -->
+Ember is a front end framework.  Someone would want to use Ember, or any other frontend framework,
+  for many reasons.  One being it helps manage app complexity.  It keeps your app seperated into
```

Comment:
> One being it helps manage app complexity.

Yes, having separated back-end and front-end systems makes it easier to maintain both of them.